### ISSUE TITLE
🐛 `ktx create` bug w/conversion of png => ktx2

### ISSUE THREAD

**Issue by Bug-Reaper** [KhronosGroup/KTX-Software#776]
#### Using some of @donmccurdy 's `ktx create` CLI recipes to try and convert equirectangular images from png to ktx2. 

---

Given the input image 2048x1024px:

![dmosh-tester](https://github.com/KhronosGroup/KTX-Software/assets/31227781/e8f4d2ae-7de7-4756-9961-47a9f2bea786)
And command:

```bash
ktx create --encode basis-lz --format R8G8B8_SRGB --assign-oetf srgb --assign-primaries bt709 dmosh-tester.png dmosh-tester.ktx2
```

I end up with malformed ktx2 that looks like this wherein original image has been shrunk, duplicated and half of new image is all black.

<img width="927" alt="Screenshot 2023-10-01 at 9 16 05 PM" src="https://github.com/KhronosGroup/KTX-Software/assets/31227781/1cac4524-bf5e-448a-b519-eb3ab8bf1a45">

If I use the other encoder, then resultant KTX2 also has same half black thing and image cannot be opened in PVRTexToolGui. I'd like to automate this type of conversion to make this process easier, any assistance appreciated.

---

I'm using an M1 mac, may or may not be relevant but don't have another machine to test with to confirm.

**Comment by MarkCallow** [comment]
This is the same bug as #727. It has been fixed in both `main` and `4.2` branches but release have not yet been made for various reasons.

**Comment by Bug-Reaper** [comment]
Okay, good to know I can use most recent main 🎀
